packages/paste-website/src/pages/utilities/flex/index.mdx

```diff
@@ -0,0 +1,353 @@
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+#### Vertical Alignment Properties
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex vAlignContent="center">
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        width="100%"
+      >
+        Left area
+      </Box>
+    </Flex>
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+        height="size10"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+#### Horizontal Alignment Properties
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex hAlignContent="center" vertical>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        width="100%"
+      >
+        Left area
+      </Box>
+    </Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="size30"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+### Composition Notes
+
+The Flex component is used to compose flexible layout experiences. Let's look at an example
+where we need to build a flexible content layout with two sidebars: its has a left sidebar,
+a content area, and a right sidebar. Here's how we’ll compose the flexible layout using Flex:
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        height="size20"
+      >
+        Left sidebar area
+      </Box>
+    </Flex>
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+        height="size20"
+      >
+        Content area
+      </Box>
+    </Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        height="size20"
+      >
+        Right sidebar area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+### When to use Flex
+
+The Flex component should be used when a flexible layout is needed with a page layout or custom component layout.
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use Flex to build responsive experiences."
+    preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Use Flex to set margins or padding within layouts and components."
+    preview={false}
+  />
+</DoDont>
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use Flex to layout your page or custom component."
+    preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Use Flex without any child elements, because this is only a container element."
+    preview={false}
+  />
+</DoDont>
+
+<DoDont>
+  <Do
```

Comment:
remove the 3rd do as you're building the grid component so lets not advertise you building your own